bears/general/IndentationBear.py

```diff
@@ -0,0 +1,201 @@
+from coalib.bears.LocalBear import LocalBear
+from bears.general.AnnotationBear import AnnotationBear
+from bears.general.AnnotationBear import starts_within_ranges
+from coalib.parsing.StringProcessing.Core import unescaped_search_for
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.results.SourceRange import SourceRange
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.AbsolutePosition import AbsolutePosition
+from coalib.results.Diff import Diff
+
+
+class IndentationBear(LocalBear):
+
+    def run(self,
+            filename,
+            file,
+            dependency_results: dict,
+            language: str,
+            language_family: str,
+            use_tabs_instead_of_spaces: bool=False,
+            indentation_width: int=4):
```

Comment:
I have this name in the `RuboCopBear` too we should document these findings so that they can be solved at once